### PR TITLE
doc(docker): update docs for docker env vars

### PR DESCRIPTION
**Pull request by opichon**
## Proposed changes

Updated docs for docker environment variables. Specifically added references for `POSTGRES_PASSWORD_FILE`, `WEBLATE_ADMIN_PASSWORD_FILE` and `WEBLATE_EMAIL_HOST_PASSWORD_FILE`.

## Checklist

- [ ] Lint and unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added documentation to describe my feature.
- [ ] I have squashed my commits into logic units.
- [ ] I have described the changes in the commit messages.

### CODE REVIEW

File: docs/admin/install/docker.rst [hunk 1]
```diff
@@ -985,6 +994,10 @@ both Weblate and PostgreSQL containers.
 
     PostgreSQL password.
 
+.. envvar:: POSTGRES_PASSWORD_FILE
+
+    Path to file containing the PostgreSQL password. use as an alternative to POSTGRES_PASSWORD.
```
Comment (orangesunny) [inline docs/admin/install/docker.rst:999]: Please do a small grammatic correction to:

*Path to the file containing the PostgreSQL password. Use as an alternative to POSTGRES_PASSWORD.*

File: docs/admin/install/docker.rst [hunk 2]
```diff
@@ -348,9 +348,18 @@ Generic settings
 
             :ref:`docker-admin-login`,
             :envvar:`WEBLATE_ADMIN_PASSWORD`,
+            :envvar:`WEBLATE_ADMIN_PASSWORD_FILE`,
             :envvar:`WEBLATE_ADMIN_NAME`,
             :envvar:`WEBLATE_ADMIN_EMAIL`
 
+.. envvar:: WEBLATE_ADMIN_PASSWORD_FILE
+
+    Sets the path to a file containing the password for the `admin` user.
```
Comment (orangesunny) [inline docs/admin/install/docker.rst:357]: please change `a` to `the` file....

File: docs/admin/install/docker.rst [hunk 3]
```diff
@@ -1114,6 +1127,14 @@ Example SSL configuration:
 
     .. seealso:: :setting:`django:EMAIL_HOST_PASSWORD`
 
+.. envvar:: WEBLATE_EMAIL_HOST_PASSWORD_FILE
+
+    Path to file containing the e-mail authentication password.
```
Comment (orangesunny) [inline docs/admin/install/docker.rst:1132]: Please do a small grammatic correction to:

*Path to the file containing the e-mail authentication password.*

**Review by orangesunny — CHANGES_REQUESTED** [review]
I will approve once the suggested small text polishing is done.

@opichon Thank you for the contribution!

File: docs/admin/install/docker.rst (hunk 1 above)
Comment (orangesunny) [reply docs/admin/install/docker.rst]: @opichon the article `the` is still missing. Can you copy-paste my suggestion?

File: docs/admin/install/docker.rst (hunk 3 above)
Comment (orangesunny) [reply docs/admin/install/docker.rst]: Thanks for the implementation!

File: docs/admin/install/docker.rst [hunk 4]
```diff
@@ -1114,6 +1127,14 @@ Example SSL configuration:
 
     .. seealso:: :setting:`django:EMAIL_HOST_PASSWORD`
 
+.. envvar:: WEBLATE_EMAIL_HOST_PASSWORD_FILE
+
+    Path to the file containing the e-mail authentication password.
+
+    .. seealso::
```
Comment (nijel) [inline docs/admin/install/docker.rst:1134]: Please remove trailing line:

```suggestion
    .. seealso::
```

(The pre-commit.ci should automatically fix, but it did timeout this time - https://results.pre-commit.ci/run/github/3559019/1619528947.7e1x1TEpRxeBxI7RoxbL4Q)

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**